src/clojure_ttt/presenter.clj

```diff
@@ -0,0 +1,29 @@
+(ns clojure-ttt.presenter
+  (use [clojure-ttt.board]))
+
+(defn- clear-console []
+  (println "\033[H\033[2J"))
+
+(defn- welcome-user []
+  (println "\033[35mWelcome! Please select a gamemode.")
+  (println "------------------------------------------"))
+
+(defn show-gamemode-options []
+  (clear-console)
+  (welcome-user)
+  (println "\033[36mType 0 to play human vs human. \n1 to play vs a computer. \n2 to watch two computers play."))
+
+(defn show-board [board]
+  (let [partitioned-board (map #(apply str %) (partition 3 board))]
```

Comment:
The `3` here is breaking encapsulation. Create a `board-size` function in your board namespace and call that instead. Your display functions should not know the size of the board.